src/ol/proj/epsg4326.js

```diff
@@ -63,7 +63,7 @@ export const PROJECTIONS = [
   new EPSG4326Projection('EPSG:4326', 'neu'),
   new EPSG4326Projection('urn:ogc:def:crs:OGC:1.3:CRS84'),
   new EPSG4326Projection('urn:ogc:def:crs:OGC:2:84'),
-  new EPSG4326Projection('http://www.opengis.net/def/crs/OGC/1.3/CRS84', 'neu'),
+  new EPSG4326Projection('http://www.opengis.net/def/crs/OGC/1.3/CRS84'),
```

Comment:
According to https://docs.opengeospatial.org/pol/09-048r5.html#_names_for_epsg_definitions, `http://www.opengis.net/def/crs/OGC/1.3/CRS84` is equivalent to the previous `CRS:84` identifier, which uses the standard `enu` axis order.

Reply:
Interesting.  I think I added this after using an early GeoServer example.  But could be remembering wrong.